HaruhiGekidouTests/SourceTests.cs

```diff
@@ -44,6 +47,52 @@ public class SourceTests
             "Tutorial_006"
         ];
 
+        public static string[] _AdvPartScriptFiles()
```

Comment:
The tiniest of nits, but generally we don't start methods with `_` in C#, so I'd just remove the underscore